Research/ResearchUI/iOS/Views/ThemeImageViewOwner.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  ThemeImageViewOwner.swift
+//  ResearchUI (iOS)
+//
+//  Copyright © 2017-2019 Sage Bionetworks. All rights reserved.
```

Comment:
was this file moved from somewhere or renamed? otherwise why does a new file have a copyright date range

Reply:
Moved the implementation into shared protocol and use extension to define the actual implementation.